### ISSUE TITLE
"Bookmarks" / "Tools" menus dont open with the keyboard

### ISSUE THREAD

**Issue by shlomif** [inbasic/bookmarks-manager#31]
Hi!

The "Bookmark \/" "Tools \/" etc. menus do not open using the keyboard - tried enter, spacebar, down arrow - no luck. Clicking the mouse opens them fine, but i want it accessible/usable. Thanks!

**Comment by inbasic** [comment]
True, they are not selectable however we have new keyboard shortcuts which basically cover the entire functionality. See FAQ 11; https://add0n.com/bookmarks-manager.html
